### DISCUSSION TITLE
LSP reports "'stdio.h'  file not found'

### DISCUSSION THREAD

**Discussion by Littlechay** [orbitalquark/textadept#551] [Q&A]
Me again with another silly questions....

I have installed the lsp module (I installed all the modules using the link on the textadept page), and configured it for ansi_C like this:
-- lsp stuff
```
local lsp = require('lsp')

lsp.server_commands.cpp = 'clangd'
lsp.server_commands.ansi_c = 'clangd'
lsp.server_commands.go = 'gopls'
```
But I get this error and further down more warning such as printf undeclared library function etc..
![image](https://github.com/user-attachments/assets/6cd20327-7445-4982-8c75-2748a406e1ce)

So my question  is: How do I tell the language server where the standard header files are located? They are in /usr/include .

Edit: I should also add that although math.h does not show a warning, as can be seen in the screenshot, that in the code I get warnings for math functions stating that math.h is not included. 

The project compiles and runs just fine.

**Comment by orbitalquark** [comment]
clangd usually needs a "compile_commands.json" file or similar to provide context: https://clangd.llvm.org/design/compile-commands. You shouldn't have to pass any flags or configuration options to clangd.
